package.json

```diff
@@ -50,6 +50,7 @@
     "minimatch": "^2.0.4"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "before": "ember-cli-deploy-redis"
```

Comment:
Agree re: the advanced config. But I think we can add these order hints as necessary to help make plugins work together out of the box.